adb-vagrantfile-tests/k8s-tests/README.md

```diff
@@ -0,0 +1,33 @@
+### Current flow
+
+
```

Comment:
What would you like me to add? About what tests are being performed at the moment?